applications/evaluation-tool/User's.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>TestDriven.io-User's</title>
+    </head>
+
+    <body>
+        <nav>
+            <header><a href="home.html">TestDriven.io</a></header>
+            <ul>
+                <li><a href="About.html">About</a></li>
+                <li><a href="User's.html">User's</a></li>
+                <li><a href="User-Status.html">User Status</a></li>
+                <li><a href="Swagger.html">Swagger</a></li>
+                <li><a href="home.html">Log Out</a></li>
+            </ul>
+        </nav>
+        <hr>
+        <h1>User's</h1>
+    </body>
+</html>
```

Comment:
Please remove the apostrophe from `User's`, including in the file name. The `s` at the end in this case denotes plural, not possessive. And apostrophes in file names can cause annoying issues.